rfcs/044-sierra-series/README.md

```diff
@@ -0,0 +1,503 @@
+      },
+     {   // 830 & 773 contain the same information
+        "title": "ACLS Humanities E-Book.",
+        "type": "Work" //or Series?
+      }
+    ]
+    ```
+- [2125597](https://search.wellcomelibrary.org/iii/encore/record/C__Rb2125597?lang=eng&suite=cobalt&marcData=Y)
+  490 with ISSN + 830 and 773. There are 1037 bibs that have the same ISSN in a series statement but
+  with different volume subfield.
+```yaml
+"partOf": [
+  {   // 490 +830 contain duplicated info
+    "title": "Methods in Molecular Biology, Methods and Protocols; 212",
+    "partOf": [
+      {
+        "id": "abcdefg",
+        "identifiers": [
+          {
+            "identifierType": {
+              "id": "issn",
+              "label": "ISSN",
+              "type": "IdentifierType"
+            },
+            "value": "1064-3745",
+            "type": "Identifier"
+          }
+        ],
+        "title": "Methods in Molecular Biology, Methods and Protocols",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  },
+  {   // 773
+    "title": "Springer eBooks",
+    "type": "Work"
+  }
+]
+```
+- [3001878](https://search.wellcomelibrary.org/iii/encore/record/C__Rb3001878?lang=eng&suite=cobalt&marcData=Y)
+  Only 830 with no 490/440 or 773
+```yaml
+"partOf": [
+    {
+        "title": "Adamson Collection Wellcome Library", // subfield $a concatenated with $p
+        "type": "Series"
+    }
+]
+```
+- [1110225](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1110225?lang=eng&suite=cobalt&marcData=Y)
+Series statement in 440 with only the title
+```yaml
+"partOf": [
+    {
+        "title": "Monographs on inorganic and physical chemistry",
+        "type": "Series"
+    }
+]
+```
+- [1204561](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1204561?lang=eng&suite=cobalt&marcData=Y) 440 with id
+```yaml
+"partOf": [
+  {
+    "title": "Brill's studies in intellectual history; v. 13",
+    "partOf": [
+      {
+        "id": "abcdefgh",
+        "identifiers": [
+          {
+            "identifierType": {
+              "id": "issn",
+              "label": "ISSN",
+              "type": "IdentifierType"
+            },
+            "value": "0920-8607",
+            "type": "Identifier"
+          }
+        ],
+        "title": "Brill's studies in intellectual history",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  }
+]
+```
+- [2301867](https://search.wellcomelibrary.org/iii/encore/record/C__Rb2301867?lang=eng&suite=cobalt&marcData=Y)
+  773 not overlapping with series and no id.
+```yaml
+"partOf": [
+    {
+        "title": "Eighteenth Century collections online",
+        "type": "Work"
+    }
+]
+```
+- [1186777](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1186777?lang=eng&suite=cobalt&marcData=Y)
+  773 with an id in subfield $w which links to another Work in our library (see example below)
+```yaml
+"partOf": [
+   {
+    "title": "Basil Hood. Photograph album; page 9",
```

Comment:
In the current implementation, the title of a partOf appears to be the name of the thing that this is part of.

In this proposal, it appears to be the name of this thing considered in the context of the other.

By reusing the field in this way, a client would have to examine other properties in order to work out what the field means.

e.g.
This means "I am a part of the work 'My Greatest Hits', which has the identifier 'baadf00d'".
```
"partOf": [{
  "title": "My Greatest Hits",
  "id": "baadf00d"
}]
```


Whereas this seems to mean "I am the part called 'Portraits of Cool People, Page 23' of 'Portraits of Cool People'"

```
"partOf":[{
  "title": "Portraits of Cool People, Page 23",
  "partOf": [{
   "title": "Portraits of Cool People",
    "id": "deadbeef"
  }]
}]
```

So an existing application that asks of a record "what are you a part of?" will currently just take partOf.title as the answer. Given this change, it would have to check whether there is a partOf.partOf, then drill into that to find it.

Perhaps this property should have a different name e.g. "partTitle"?

Reply:
Actually, I think I'm a bit wrong here.

What this is saying is I am a part of page 9 of the Basil Hood Photo Album which is part of the Basil Hood Photo Album.

In that case, nested partOf makes sense, semantically.  However, I'm still slightly concerned about the change of interface here.

Reply:
It's notable that "page 9" is not something with an identifier in either direction. The 773s refer to the id of the album, and the 774s refer to the ids of each photo.

Reply:
Another thing that worries me about nesting partOf objects (from a client PoV) is "where will it end?"

Were I an external consumer of this API, I would want to know how far I need to recurse down the nest until I get to the thing I want.

From an internal perspective, it looks like we would only make this one level deep and with only one entry in the nested one, so _we_ could write things like partOf[n].partOf[0] and know we are at the terminus.

Someone else could not (or they could, but would not necessarily be confidently correct).  They would have to loop and recurse until some condition is met (no more partOfs, a partOf with an id?)

Similarly, if we do nest them, would a consumer have an expectation that the nesting should be comprehensive?  e.g. "article A is part of vol2 of journal B which is in collections D, E and F, collection D is in ..."